docs/plugins/test_deploy.md

```diff
@@ -119,14 +116,15 @@ development process, see the [napari hub GitHub’s
 Wiki](https://github.com/chanzuckerberg/napari-hub/wiki).
 
 If you want your plugin to be available on PyPI, but not visible on the napari
-hub, you can add a `.napari/config.yml` file to the root of your repository with
-a visibility key. For details, see the [customization
+hub, you can add a `.napari-hub/config.yml` file to the root of your repository
```

Comment:
Here I'm torn as currently the only option for updating visibility on the hub is to add this field (or remove your framework classifier). However, we really want to encourage adoption of this field from the manifest instead. We need to add this to `napari`, and the napari hub should also change their logic to read this field instead. We can open PRs to plugins using `config.yml` as needed, to avoid a lengthy deprecation step. I think for now this change is fine though.

Reply:
I'm ambivalent about this change at this point, since regardless of what is said here, it's likely to change based on the discussion you referenced. The only advantage I guess is that it makes things consistent with the hub guide.

Reply:
@kandarpksk I've opened [napari/#5879](https://github.com/napari/cookiecutter-napari-plugin/issues/160) to track the implementation of the `visibility` field in the plugin installer. I suggest we leave this bit of the docs as is until that's done, when we can update this section to refer to the manifest `visibility` field.

Reply:
I've excluded this change in the latest commit, and will update the PR title appropriately!